core/src/main/java/org/fao/geonet/api/records/attachments/CMISStore.java

```diff
@@ -54,23 +54,30 @@
 import java.io.*;
 import java.nio.file.*;
 import java.util.*;
+import java.util.regex.Pattern;
 import javax.annotation.Nullable;
 import javax.resource.NotSupportedException;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 public class CMISStore extends AbstractStore {
 
     private Path baseMetadataDir = null;
 
     @Autowired
-    CMISConfiguration CMISConfiguration;
+    CMISConfiguration cmisConfiguration;
+
+    @Autowired
+    CMISUtils cmisUtils;
 
     @Override
     public List<MetadataResource> getResources(final ServiceContext context, final String metadataUuid,
                                                final MetadataResourceVisibility visibility, String filter, Boolean approved) throws Exception {
         final int metadataId = canDownload(context, metadataUuid, visibility, approved);
         final SettingManager settingManager = context.getBean(SettingManager.class);
```

Comment:
Can't be autowired?

To review as there are other places in the store beans that use similar code to retrieve the `SettingManager`.

Reply:
I believe it was done that way because it was copied from the other code.

Yes we can correct this with another PR.  I don't think I should fix it in this PR as it would make it confusing to back port it to 3.12.x since this is code that is being ported from 3.12.x.